### ISSUE TITLE
Module for download templates

### ISSUE THREAD

**Issue by Jalkhov** [jamstackpy/jamstack#2]
I have developed an interface for downloading template assets, however at the last moment I realised that it might be too unnecessary xD Since a project will only be compatible with the respective assets of its template, and it doesn't make much sense to give the user the option to choose between all the available ones, the interface looks like this:

![Screenshot_2](https://user-images.githubusercontent.com/36505934/130708353-c5d044ff-056d-4390-9bf4-3c94b517e8a8.png)

I managed to integrate the module with Jamstack and it is called through the `--jamdo` option in the `static.py` file of each project. So, taking into account the above, I see as a better way that instead of showing the interface, **the command automatically starts downloading the assets** @Abdur-rahmaanJ What do you say?

**Comment by Abdur-rahmaanJ** [comment]
Normally @Jalkhov what NLTK does is to zip all files into zip and to download and extract the zip. Like template and assets

**Comment by Abdur-rahmaanJ** [comment]
It downsloads them to an internal folder in the package. we can also have cookie cutter versions, zip them, download and create based on them

**Comment by Jalkhov** [comment]
@Abdur-rahmaanJ 

Well, what I had implemented was something different, I uploaded the assets of each template to a repo https://github.com/Jalkhov/jamstack-templates, and what the downloader does is to download the files from the respective folder, to the dist/assets folder of the project, it works quite well, and so in case you want to add more templates just add them to the respective repo. I'm going to make a PR for you to see better because I don't know if I'm explaining myself well.

In any case, I was thinking that instead of creating a special command to download the assets, that they should be downloaded automatically when creating a new project.
